pkg/webui/components/shortcut-panel/shortcut-item/shortcut-item.styl

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.shortcut
+  transition: $ad.m background ease-in, $ad.m display ease-out
+  text-decoration: none
+  height: 10.7rem
+  border-radius: $br.l
+  background: var(--c-bg-brand-normal)
+  color: var(--c-text-neutral-min)
+  font-size: $fs.m
+  font-weight: $fwv2.semibold
+
+  span.icon
+    font-size: 2rem
+    display: block
+
+  span.add-icon
+    font-size: 2.5rem
+    display: none
+
+  .title
+    display: block
+
+  &:hover
+    cursor: pointer
```

Comment:
Is `cursor: pointer` necessary here? The component is already wrapped around an `a` tag, via `<Link />`.